cdap-docs/user-guide/source/tutorials/logistic.rst

```diff
@@ -0,0 +1,237 @@
+
+`set-type normalized_merchant_fraud_risk Double`
+`set-type normalized_time_since_last_transaction Double`
+`set-type normalized_days_till_expiration Double`
+`set-type normalized_change_in_merchant_sales Double`
+`set-type normalized_transaction_time Double`
+`set-type Amount Double`
+`set-type Class Double`
+
+Training: Deploying the Model in a Pipeline
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In the upper right hand corner, click "Create Pipeline." Choose Batch.
+
+You will see the following configuration appear on your screen.
+
+.. figure:: /_images/tutorials/logistic/start_pipeline.jpeg
+	:figwidth: 100%
+	:width: 500px
+	:align: center
+	:class: bordered-image
+
+Since you are training a Logistic Regression model, you now want to add a Logistic Regression plugin to the canvas. You will need to download this plugin, which availabe on the Cask Market. 
+
+Choose "Cask Market" in the global navigation bar. Find "Logistic Regression Analytics" in the application menu and deploy the plugin.
+
+Now, save the pipeline as "LogisticRegressionTrainer".
+
+.. figure:: /_images/tutorials/logistic/save_trainer.jpeg
+	:figwidth: 100%
+	:width: 500px
+	:align: center
+	:class: bordered-image
+
+Refresh the page, and under the "Sink" tab in the left side menu, you will find the "Logistic Regression Trainer". Add this plugin the canvas, and connect it to the "Wrangler" stage. You will see the configuration below:
+
+.. figure:: /_images/tutorials/logistic/first_version_trainer.jpeg
+	:figwidth: 100%
+	:width: 500px
+	:align: center
+	:class: bordered-image
+
+Looking at the canvas, you will see that several stages have small yellow numbers displayed in the upper right hand corner. These values tell you how many fields need to be configured before the pipeline can be deployed and run.
+
+Open the `File` stage and rename it (by changing the label and reference name) to "CreditCardSource." Similarly, rename "Wrangler" to "CreditTransform" and "LogisticRegressionTrainer" to "FraudTrainerSink." 
+
+In the Trainer configuration, specify `Label Field` to be Class. This is the field that you Logistic Regression model will predict. Also, give `FileSet Name` the value "FraudModel," which is the fileset in which the trained model will be stored.
+
+.. figure:: /_images/tutorials/logistic/trainer_config.jpeg
+	:figwidth: 100%
+	:width: 500px
+	:align: center
+	:class: bordered-image
+
+Your configured pipeline will appear as the pipeline below.
+
+.. figure:: /_images/tutorials/logistic/configured.jpeg
+	:figwidth: 100%
+	:width: 500px
+	:align: center
+	:class: bordered-image
+
+To test your pipeline configuration, click `Preview` in the upper right hander corner, then `Run`. Opening the LogisticRegressionTrainer, you will be able to see the records that have flown through in the `Preview` tab.
```

Comment:
May be we could add `Once the preview is complete you can open the LogisticRegressionTrainer and see the records that...`